dex/testing/loadbot/mantle.go

```diff
@@ -734,10 +734,11 @@ func newBotWallet(symbol, node, name string, port string, pass []byte, minFunds,
 			Type:    "dogecoindRPC",
 			AssetID: dogeID,
 			Config: map[string]string{
-				"walletname":  name,
-				"rpcuser":     "user",
-				"rpcpassword": "pass",
-				"rpcport":     port,
+				"walletname":   name,
+				"rpcuser":      "user",
+				"rpcpassword":  "pass",
+				"rpcport":      port,
+				"feeratelimit": "40000",
```

Comment:
Do we need this?

Reply:
Apparently. It doesn't use the correct default feeratelimit for doge. It's different for doge correct? Should I look into why? I probably should.